client/src/components/common/ClickToPlayAlbum.tsx

```diff
@@ -0,0 +1,193 @@
+import styled from "@emotion/styled";
```

Comment:
This looks like it's mainly a copy of the functionality of the ClickToPlay component? I think we shouldn't duplicate components like that, if it's just the style that's different. It creates more places for bugs to exist that we have to squash in multiple places.